holoviews/streams.py

```diff
@@ -391,6 +392,77 @@ def hashkey(self):
         return {'hash': uuid.uuid4().hex}
 
 
+class DataFrameStream(DataStream):
+    """
+    DataFrameStream provides an adaptor to attach streamz
+    StreamingDataFrame to a HoloViews stream. The stream will accumulate
+    a DataFrame up to the specified ``backlog`` of rows. The accumulated
+    dataframe is then made available via the ``data`` parameter.
+
+    Optionally a chunksize may be supplied which will split received
+    data into smaller chunks resulting in smoother plot updates when
+    large numbers of samples are received.
+    """
+
+    def __init__(self, sdf, backlog=1000, chunksize=None, **params):
+        try:
```

Comment:
I guess backlog and chunksize can't be Parameters because Streams use Parameters for their own special purposes?

Reply:
That's right, although I could filter them out. @jlstevens might want to chime in.

Reply:
They certainly seem like they ought to be Parameters, so that they can be set at the class or instance level as appropriate.